lib/nimble.phx.gen.template/addons/github.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Nimble.Phx.Gen.Template.Addons.Github do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @versions %{
+    otp_version: "23.0.2",
+    elixir_version: "1.10.4"
+  }
+
+  @impl true
+  def do_apply(%Project{} = project, opts) when is_map_key(opts, :github_template) do
+    generate_github_template()
+
+    project
+  end
+
+  @impl true
+  def do_apply(%Project{} = project, opts) when is_map_key(opts, :github_action) do
+    generate_github_action()
+
+    project
+  end
+
+  def generate_github_template?(),
```

Comment:
I think this function (`generate_github_action?` also) belongs to `lib/nimble.phx.gen.template/template.ex`. So this module only responsible for generating stuff and let the `Template` module handle about a decision 🙌